dwave/system/temperatures.py

```diff
@@ -249,373 +391,741 @@ def maximum_pseudolikelihood_temperature(bqm=None,
 
         https://www.jstor.org/stable/25464568
 
-    '''
-
-    T_estimate = 0
-    T_bootstrap_estimates = np.zeros(num_bootstrap_samples)
-
-    #Check for largest local excitation in every sample, and over all samples
-    if site_energy is None:
-        if bqm is None or sampleset is None:
-            raise ValueError('site_energy can only be derived if both'
-                             'bqm and sampleset are provided as arguments')
-        site_energy = effective_field(bqm,
-                                      sampleset,
-                                      current_state_energy = True)
-    max_excitation = np.max(site_energy[0],axis=1)
-    max_excitation_all =  np.max(max_excitation)
-    if max_excitation_all <= 0:
-        #There are no local excitations present in the sample set, therefore
-        #the temperature is estimated as 0. 
-        pass
+    """
+    if T_guess:
+        x0 = [-1 / T_guess]
+    else:
+        x0 = None
+
+    if optimize_method == "bisect":
+        bisect_bracket = [-1 / T_bracket[i] for i in range(2)]
+
+        if not 0 <= T_bracket[0] < T_bracket[1]:
+            raise ValueError("Bad T_bracket, must be positive ordered scalars.")
+
+        if x0 is not None and (x0[0] < bisect_bracket[0] or x0[0] > bisect_bracket[1]):
+                raise ValueError(f"x0 {x0[0]} and T_bracket {T_bracket} are inconsistent")
+
+        kwargs_opt = {"bisect_bracket": bisect_bracket}
```

Comment:
Sorry for breaking the tests. I think this is the culprit.
```suggestion
        kwargs_opt = {"bracket": bisect_bracket}
```